package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
     "name": "@devtron-labs/devtron-fe-common-lib",
-    "version": "v0.0.32",
+    "version": "0.0.39-beta-1",
```

Comment:
Actions were failing as we are using npm ci which requires the lock file to list the dependencies to be installed.